src/modules/Search.js

```diff
@@ -39,11 +39,7 @@ module.exports = function (searchService, downloadService, options, client) {
     const filesByUser = this.filterResult.filter(res);
     this.checkEmptyResult(filesByUser);
 
-    if (this.showPrompt) {
-      this.showResults(filesByUser);
-    } else {
-      process.exit(0);
-    }
+    this.showResults(filesByUser);
```

Comment:
Hey @hejops, can you refactor it into two distinct methods please? (instead of having `showResults` doing two things).
Thanks a lot!